lib/menubar/init.lua

```diff
@@ -132,6 +132,7 @@ local function load_count_table()
                 count_table[name] = count
             end
         end
+        count_file:close()
     end
```

Comment:
but shouldn't we check then the existance of that file before calling :lines on it?